code/script.js

```diff
@@ -1,19 +1,133 @@
+const foodChoice = parseInt(
+  prompt(
+    `What are you craving today?
+    1 - Pizza
+    2 - Pasta
+    3 - Salad
+    Please choose the number of what you would like to order.`
+  )
+)
+console.log(foodChoice)
+
+let selectedFood = ""
+
+if (foodChoice === 1) {
+selectedFood = "pizza"
+} else if (foodChoice === 2) {
+  selectedFood = "pasta"
+} else if (foodChoice === 3) {
+  selectedFood = "salad"
+} else {
+  alert(`We can't identify what type of food you want to order. Please choose a valid number.`)
+  process.exit(1)
+}
+
+alert(
+  `You have chosen to order a delicious ${selectedFood}!`
+)
 
 // Step 3 - Subtype choice
-// Your code goes here
+//Using the conditional statement switch method for the customer to choose what dish they want depending on what food they selected previously
+let subMenu = ""
+let dishName = ""
+let selectedDish = ""
+
+switch (selectedFood) {
+  case "pizza": 
+    subMenu = prompt(`What dish would you like to have?
+  1 - Vesuvio
+  2 - Capricciosa
+  3 - Veggie Delight
+  Please choose the number of what you would like to order.`)
+    dishName = ["Vesuvio", "Capricciosa", "Veggie Delight"]
+    break
+  case "pasta": 
+    subMenu = prompt(`What dish would you like to have?
+  1 - Bolognese
+  2 - Carbonara
+  3 - Frutti di Mare
+  Please choose the number of what you would like to order.`)
+    dishName = ["Bolognese", "Carbonara", "Frutti di Mare"]
+    break
+  case "salad": 
+    subMenu = prompt(`What dish would you like to have?
+  1 - Caesar salad
+  2 - Greek salad
+  3 - Cobb salad
+  Please choose the number of what you would like to order.`)
+    dishName = ["Caesar salad", "Greek salad", "Cobb salad"]
+    break
+  default:
+    alert(`We can't identify what dish you want to order. Please choose a valid number.`)
+    process.exit(1)
+}
+
+selectedDish = dishName[subMenu - 1] 
+
+console.log(dishName)
+
+alert(
+  `You have chosen to order a yummy ${selectedDish}!`
+)
 
 // Step 4 - Age
-// Your code goes here
+//Using a prompt to ask for the age of the customer in order to decide a price
+const guestAge = parseInt(
+  prompt(`To estimate how big your appetite is - if you're a child or an adult, please tell us your age.`)
+)
+
+let orderMessage = ""
+
+if (guestAge <= 12) {
+  orderMessage = `You have ordered a small ${selectedDish}. That will be 100 SEK. Are you happy with this order?`
+} else if (guestAge >= 13) {
+  orderMessage = `You have ordered a large ${selectedDish}. That will be 130 SEK. Are you happy with this order?`
+} else {
+  alert(`We can't identify your age. Remember - it's just a number! Please enter how old you are below.`)
+  process.exit(1)
+}
 
 // Step 5 - Order confirmation
-// Your code goes here
+//Using a prompt to ask the customer to accept or decline their order which ends with an alert confirming the order
+const orderConfirmation = parseInt(
+  prompt(`${orderMessage}
+    Enter a number to confirm or decline your order
+    1 - Confirm
```

Comment:
Great to see you using Template Literals in this prompt message, so that you could get the orderMessage right in there without a fuss!